mysql-test/suite/mariabackup/MDEV-31042_1.opt

```diff
@@ -0,0 +1,7 @@
+--innodb-encryption-rotate-key-age=2
+--innodb_encrypt_tables=ON
+--innodb-encryption-threads=4
+--innodb-tablespaces-encryption
+--plugin-load-add=$FILE_KEY_MANAGEMENT_SO
+--loose-file-key-management
+--loose-file-key-management-filename=$MYSQL_TEST_DIR/std_data/logkey.txt
```

Comment:
There is a missing newline character at the end of the file.

I would like test names to be descriptive, such as `incremental_encrypted`. Instead of creating new test files, could that existing test be extended?